docs/TuxxAnimations.md

```diff
@@ -222,25 +222,25 @@ The keys define the transition state and the values are objects that describe th
 The customClassName parameter requires a string that gives your component a custom className property so that it can be accessed easily via CSS or jQuery etc.. This way when your component is converted to a ReactElement, it is still accessible via this customClassName property:
 
 ```javascript
-  <div class="zoom" data-reactid=".0.2.2.1.$=10:0"></div>
+  <div class="scaleUp" data-reactid=".0.2.2.1.$=10:0"></div>
```

Comment:
Because eventually, I think it makes sense to deprecate Zoom and Fly. We can leave it in for now if people like using it, but Scale Up or Down do a much better job of describing the animation and stick with the other folder names in being descriptive of the actions within them. I couldn't really tell you what a zoom or fly do, but Fade Up, Scale Down, Slide Right; these make more sense semantically.